app/components/comment.hbs

```diff
@@ -5,6 +5,12 @@
     <div class="post-description">on <span class="date">{{format-date @comment.createdAt}}</span>
     </div>
   </div>
+  <PaperButton
+  class="delete-icon {{if (eq @comment.user @getMe) "active"}}"
+  @iconButton={{true}}
+  {{on "click" @deleteCommentbyId}}>
```

Comment:
`Assertion Failed: You must pass a function as the second argument to the `on` modifier`